state/core_access.go

```diff
@@ -459,6 +524,53 @@ func (ca *CoreAccessor) QueryRedelegations(
 	})
 }
 
+func (ca *CoreAccessor) LastPayForBlob() int64 {
+	ca.lock.Lock()
+	defer ca.lock.Unlock()
+	return ca.lastPayForBlob
+}
+
+func (ca *CoreAccessor) PayForBlobCount() int64 {
+	ca.lock.Lock()
+	defer ca.lock.Unlock()
+	return ca.payForBlobCount
+}
+
+func (ca *CoreAccessor) markSuccessfulPFB() {
+	ca.lock.Lock()
+	defer ca.lock.Unlock()
+	ca.lastPayForBlob = time.Now().UnixMilli()
+	ca.payForBlobCount++
+}
+
+func (ca *CoreAccessor) setMinGasPrice(minGasPrice float64) {
+	ca.lock.Lock()
+	defer ca.lock.Unlock()
+	ca.minGasPrice = minGasPrice
+}
+
+func (ca *CoreAccessor) getMinGasPrice() float64 {
+	ca.lock.Lock()
+	defer ca.lock.Unlock()
+	return ca.minGasPrice
+}
+
+// QueryMinimumGasPrice returns the minimum gas price required by the node.
+func (ca *CoreAccessor) queryMinimumGasPrice(
```

Comment:
This could be exported, so we can expose it through the RPC

Reply:
We can export min gas price for sure. Care for an issue?

Reply:
yes